website/docs/module_kitsu.md

```diff
@@ -53,5 +53,5 @@ There are four settings available:
 
 ## Q&A
 ### Is it safe to rename an entity from Kitsu?
-Absolutely! Entities are linked by their unique IDs between the two databases.  
```

Comment:
The labeler had revealed you. Trying to smuggle some Kitsu changes? ;)

Reply:
This might have been a merge conflict that didn't commit correctly. I run into that from time to time where it doesn't correctly commit the merge commit due to the pre commit hooks. Will investigate next week.

Reply:
Had another look - apparently I did end up committing two changed line endings (new-lines). I guess we can ignore it since they don't seem like breaking changes and cosmetically fit with the code style OP is using. Thoughts?

Reply:
Testing